apps/buildog/src/app/organizations/[organizationId]/layout.tsx

```diff
@@ -21,16 +21,16 @@ export default function Layout({
 }) {
   const { organizationId } = params;
 
+  const organizations = await getOrganizations();
+
+  const currentOrganization = organizations.find((org) => org.id === organizationId);
+
   return (
     <div className="flex w-full">
-      {/* Pass organizationName to Sidebar */}
       <Sidebar className="w-[300px] border-r" organizationId={organizationId} />
       <div className="flex flex-col w-full">
-        <Appbar />
-        <div className="flex-grow p-5 mx-auto overflow-auto w-full">
-          {/* Render the children (nested pages) */}
-          {children}
-        </div>
+        <Appbar organizations={organizations} currentOrganization={currentOrganization} />
```

Comment:
Can you manage organizations and currentOrganization in <Appbar/> component? Appbar component will use other pages as well. It would be better we can get the organizations in the component.